assets/loading/loading.pack

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Jak się robi takie rzeczy? Jest jakiś generator? Czy to znalazłeś gdzieś :P

Reply:
Taki plik jest akurat był w jakiejś templatce do robienia ekranów ładowania, ale ogólnie to robi się to przez TexturePacker. Nie jestem do końca pewny, ale gdzieś w internecie jest jar, albo pobiera się go w gdx-setup.